src/framework/components/audiosource/audiosource_component.js

```diff
@@ -101,6 +109,35 @@ pc.extend(pc, function () {
             }
         },
 
+        /**
+         * @function
+         * @name pc.AudioSourceComponent#getCurrentTime
+         * @description Get the current time of the playback.
+         */
+        getCurrentTime: function () {

```

Comment:
I think the API here should match the animation component:
https://github.com/playcanvas/engine/blob/master/src/framework/components/animation/animation_component.js#L268
In other words, rather than functions, a property would be better.